app/models/routable_artefact.rb

```diff
@@ -0,0 +1,87 @@
+require 'plek'
+require 'gds_api/router'
+
+class RoutableArtefact
```

Comment:
Okay! I'd probably be in favour of only copying over what's strictly needed as it would make it easier to read/review and remove any unnecessary code. But no reason to hold up this PR!